src/celeritas/phys/PhysicsTrackView.hh

```diff
@@ -236,6 +242,17 @@ CELER_FUNCTION void PhysicsTrackView::dedx_range(real_type range)
     this->state().dedx_range = range;
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Set the range properties for multiple scattering.
+ *
+ * These values will be calculated at the first step in every tracking volume.
+ */
+CELER_FUNCTION void PhysicsTrackView::msc_range(MscRange msc_range)
```

Comment:
Since the argument is larger than a pointer/double, take the value by const reference.

Reply:
Done.